matter/wifi/rs911x/wfx_rsidev.c

```diff
@@ -60,12 +60,12 @@ static void wfx_host_gpio_init(void)
   CMU_ClockEnable(cmuClock_GPIO, true);
 
   // Configure WF200 reset pin.
-  GPIO_PinModeSet(SL_WFX_HOST_PINOUT_RESET_PORT, SL_WFX_HOST_PINOUT_RESET_PIN, gpioModePushPull, 0);
+  GPIO_PinModeSet(SL_WFX_HOST_PINOUT_RESET_PORT, SL_WFX_HOST_PINOUT_RESET_PIN, gpioModePushPull, CLEAR_BUFFER);
```

Comment:
The define is inconsistent with other PinModeSet. Should be `PINOUT_CLEAR`

Reply:
done